src/engine/server/sv_init.cpp

```diff
@@ -52,6 +52,9 @@ static Cvar::Cvar<std::string> cvar_pakname(
 	"pakname", "pak containing current map", Cvar::SERVERINFO | Cvar::ROM, "");
 static Cvar::Cvar<std::string> sv_paks(
 	"sv_paks", "currently loaded paks", Cvar::SYSTEMINFO | Cvar::ROM, "");
+static Cvar::Cvar<std::string> cvar_abuse(
+	"abuse", "contact information to reach out the server admin when needed (forum or chat nickname, mail address…)",
```

Comment:
`out the server admin` -> `out t the server admin`

Reply:
Thanks, if some have better wording for the whole sentence, it's welcome too.

Reply:
I fixed the string.